pipelines/nextflow/modules/database/db_factory.nf

```diff
@@ -15,6 +15,7 @@
 
 process DB_FACTORY {
     label 'local'
+    containerOptions '--bind ./:/output'
```

Comment:
Out of curiosity, what does this do?

Reply:
Nothing apparently. It was defined when Matthieu created the pipeline but removing it made no difference. Its will go